torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -68,10 +68,14 @@ class _X86InductorQuantizationAnnotation(QuantizationAnnotation):
     torch.ops.aten.flatten.using_ints,
 }
 
-
+embedding_ops: Set = {
+    torch.ops.aten.embedding_bag.padding_idx,
+}
 # Operations support the int8 data type and exclude operations such as conv and linear.
 # A superset of int8_in_int8_out_ops_pt2e incorporating additional operators.
-quantizable_ops_pt2e = copy.deepcopy(int8_in_int8_out_ops_pt2e)
+quantizable_ops_pt2e = copy.deepcopy(int8_in_int8_out_ops_pt2e).union(
+    copy.deepcopy(embedding_ops)
```

Comment:
If we always annotate weight of `embedding_ops`. We don't need to add it into `quantizable_ops_pt2e ` as the comment above.